skins/laika/src/components/App.vue

```diff
@@ -30,6 +26,7 @@
 import Vue from 'vue';
 import Buefy from 'buefy';
 import Header from '@/components/layout/Header.vue';
+import Hero from '@/components/layout/Hero.vue';
```

Comment:
Why `Hero` ?

Reply:
That was the outcome of a tiny naming discussion with Gabriel (https://en.wikipedia.org/wiki/Hero_image), alternatively we can also just call it `Headline` or `Tagline` if you think either of those would be better.

Reply:
Yeah you know my opinion of naming things with words which first meaning is not that.
I'd be in favor of either headline or tagline.

Reply:
+1 for Tagline. "Hero", while technically correct, looks like Web-Designery Jargon to me and was more of a tongue-in-cheek proposal.